wallet-webapps-common/src/main/webapp/css/main.less

```diff
@@ -1099,3 +1099,9 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
     }
   }
 }
+
+#WalletNotificationAlert {
+  .v-icon {
+    align-self: center !important;
```

Comment:
useless change, we already defined re-usable toast notif without custom CSS definition

Reply:
yes indeed, the already built exoNotificationAlert has been used .